static/set_proxy.sh

```diff
@@ -30,17 +30,11 @@ if hash git 2>/dev/null; then
   git config --global --unset http.proxy
   git config --global --unset https.proxy
 fi
-if hash docker 2>/dev/null; then
```

Comment:
Still not there?

Reply:
I have added that part in the reset_proxy() function.

Reply:
Actually having two versions of a script in the same repository is leading to confusion. I am suggesting we should have one repository for the script and another one for the webapp.